akka-distributed-data/src/main/scala/akka/cluster/ddata/DurableStore.scala

```diff
@@ -311,4 +322,30 @@ final class LmdbDurableStore(config: Config) extends Actor with ActorLogging {
     }
   }
 
+  def dbDelete(keys: Set[KeyId]): Unit = {
+    val t0 = System.nanoTime()
```

Comment:
only system nano time for t0 if `log.isDebugEnabled` as well?

Reply:
👍 3b774e15ac